### PR TITLE
Move ESC telemetry logging

### PR DESCRIPTION
**Pull request by amilcarlucas**

### CODE REVIEW

File: libraries/AP_Logger/LogStructure.h [hunk 1]
```diff
@@ -1513,12 +1466,9 @@ LOG_STRUCTURE_FROM_CAMERA \
       "SIM","QccCfLLffff","TimeUS,Roll,Pitch,Yaw,Alt,Lat,Lng,Q1,Q2,Q3,Q4", "sddhmDU????", "FBBB0GG????" }, \
     { LOG_TERRAIN_MSG, sizeof(log_TERRAIN), \
       "TERR","QBLLHffHH","TimeUS,Status,Lat,Lng,Spacing,TerrH,CHeight,Pending,Loaded", "s-DU-mm--", "F-GG-00--" }, \
-    { LOG_ESC_MSG, sizeof(log_Esc), \
-      "ESC",  "QBeffcfcf", "TimeUS,Instance,RPM,Volt,Curr,Temp,CTot,MotTemp,Err", "s#qvAOaO%", "F-B--BCB-" }, \
+    LOG_STRUCTURE_FROM_ESC_TELEM \
     { LOG_CSRV_MSG, sizeof(log_CSRV), \
       "CSRV","QBfffB","TimeUS,Id,Pos,Force,Speed,Pow", "s#---%", "F-0000" }, \
-    { LOG_CESC_MSG, sizeof(log_CESC), \
```
Comment (andyp1per) [inline libraries/AP_Logger/LogStructure.h:1520]: This is missing from the target

**Review by andyp1per — CHANGES_REQUESTED** [review]

File: libraries/AP_Logger/LogStructure.h (hunk 1 above)
Comment (amilcarlucas) [reply libraries/AP_Logger/LogStructure.h:1520]: I removed it, nobody calls it.

**Comment by amilcarlucas** [comment]
Can you "approve and run" the workflows on this PR?

**Comment by amilcarlucas** [comment]
Thanks, All CI tests pass.

**Review by andyp1per — APPROVED** [review]

**Pull request merged**